tor/core/validation.py

```diff
@@ -39,13 +67,17 @@ def verified_posted_transcript(post, config):
     # get source link, check all comments, look for root level comment
     # by the author of the post and verify that the key is in their post.
     # Return True if found, False if not.
```

Comment:
This seems like a much more concise way of expressing the same concepts as in the docstring above. Maybe this should replace the docstring?

Reply:
I feel like the description that's currently in the docstring explains different things. I've moved this bit up there, though.